runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -282,6 +289,11 @@ public void close() throws Exception
         }
     }
 
+    private void drain()
+    {
+        dispatchers.forEach(d -> d.drain());
```

Comment:
```suggestion
        dispatchers.forEach(DispatchAgent::drain);
```

Reply:
Also suggest we just inline this, since we don't need a separate `Engine.drain()` method.